### ISSUE TITLE
Run Ostrom's playbook

### ISSUE THREAD

**Issue by whit537** [gratipay/inside.gratipay.com#1168]
From [slack](https://gratipay.slackarchive.io/gratipay/page-100/ts-1505829424000021):

> I’m thinking about how to make a real dent in the problem (the problem: https://gratipay.news/open-source-captures-almost-none-of-the-value-it-creates-9015eb7e293e).
> The problem is the tragedy of the open source commons.
> Appropriators appropriating without supporting provisioners provisioning.
> “Appropriators” and “provisioners” are Ostrom’s terms for consumers and producers.
> If we really believe that there is an open source tragedy of the commons happening,
> and we are really trying to solve that problem,
> then Ostrom is our blueprint.
> https://twitter.com/Gratipay/status/908710955265220611
> boundaries, adaptation, democracy
> monitoring, sanctions, courts
> recognition, nesting
> tl;dr I’m not sure how this problem gets solved without some “toothy” institution.
> Basically https://github.com/ossfriendly/open-source-sponsors but with more oomph behind it.
> One Ring to Rule Them All.
> A single institution that accepts dues from member organizations in exchange for a “We pay for open source” stamp of approval.
> The stamp of approval needs to matter in hiring (and possibly purchasing) for this to work.
> A club.
> An association.
> A cabal.
> A coalition.
> A cooperative.
> A trade association.
> An organization.
> An institution.
> An entity.
> A collective.
> A group.
> A bloc.
> A fellowship.
> A pact.
> A society.
> A consortium!
> A consortium.

**Comment by whit537** [comment]
"Unite the tribes"
